biz.aQute.bndlib/src/aQute/bnd/osgi/FileResource.java

```diff
@@ -51,7 +52,7 @@ public String toString() {
 	}
 
 	public void write(OutputStream out) throws Exception {
-		copy(this, out);
+		Files.copy(file.toPath(), out);
```

Comment:
Two reasons:

1. Files.copy() may use native code to perform the copy operation
2. FileResource.copy() is a static method and cannot be overriden